CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
```

Comment:
Please update the `## Unreleased`, there are a few merges after the release `v12.0.0` was cut

Reply:
Thanks @jennyhliu! I've updated the changelog to match the entries from the 12.0.0 release: https://github.com/nasa/cumulus/pull/2966/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed

Everything else is moved to "Unreleased".

https://github.com/nasa/cumulus/pull/2971/commits/b25c258430661d611c398055c0b5d2c4f555b9c4

Reply:
And just as I did that there was another merge to master 😄 https://github.com/nasa/cumulus/pull/2971/commits/b88db6134ecc6cabf5b32eeadb227bf328a78f1f